test/gui/shared/scripts/pageObjects/AccountConnectionWizard.py

```diff
@@ -0,0 +1,35 @@
+import names
+import squish
+from helpers.SetupClientHelper import getClientDetails
+
+
+class AccountConnectionWizard():
+    SERVER_ADDRESS_BOX = names.leUrl_OCC_PostfixLineEdit
+    NEXT_BUTTON = names.owncloudWizard_qt_passive_wizardbutton1_QPushButton
+    USERNAME_BOX = names.leUsername_QLineEdit
+    PASSWORD_BOX = names.lePassword_QLineEdit
+    SELECT_LOCAL_FOLDER = names.pbSelectLocalFolder_QPushButton
+    DIRECTORY_NAME_BOX = names.fileNameEdit_QLineEdit
+    CHOOSE_BUTTON = names.qFileDialog_Choose_QPushButton
+    CONNECT_BUTTON = names.owncloudWizard_qt_passive_wizardbutton1_QPushButton
```

Comment:
what about having the full names of the selectors just here and not in the `names` class?